certified-connectors/Data247logic/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3474 @@
+            "x-ms-summary": "Data247.com API Key"
+          },
+          {
+            "name": "phone",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Phone",
+            "x-ms-summary": "Phone"
+          }
+        ]
+      }
+    },
+    "/v3.0/TPC/": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "response": {
+                  "type": "object",
+                  "properties": {
+                    "status": {
+                      "type": "string",
+                      "description": "status"
+                    },
+                    "results": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "phone": {
+                            "type": "string",
+                            "description": "phone"
+                          },
+                          "spam_risk": {
+                            "type": "string",
+                            "description": "spam_risk"
+                          },
+                          "reason": {
+                            "type": "string",
+                            "description": "reason"
+                          }
+                        }
+                      },
+                      "description": "results"
+                    }
+                  },
+                  "description": "response"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Trust247 (Phone Number)",
+        "operationId": "Trust247-Phone-Number",
+        "description": "Trust247 (Phone Number) takes phone number as input and returns whether it's on a list of SPAM callers.",
+        "parameters": [
+          {
+            "name": "key",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Data247.com API Key",
+            "x-ms-summary": "Data247.com API Key"
+          },
+          {
+            "name": "phone",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "phone",
+            "x-ms-summary": "phone"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {},
+  "security": [],
+  "tags": [],
+  "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": "https://www.data247.com/"
+      },
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://www.data247.com/privacy"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "Data Services;Identity Check;Fraud Check"
```

Comment:
@Data247logic Custom categories are not allowed, Please go thru the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#adding-metadata) and add maximum two categories from the given list.

Reply:
fixed.